src/lib/apiGraphQL.js

```diff
@@ -230,11 +230,12 @@ const operationsDoc = `
 
   mutation CreateOpenSaucedGoalsRepo {
     gitHub {
-      createRepository(
+      cloneTemplateRepository(
```

Comment:
This is a really good idea :+1: 

The same method is used in the GitHub Education Platform to create repos for students!